src/default_event_handler.ts

```diff
@@ -0,0 +1,38 @@
+import type { EventHandler } from "./types";
+
+export class DefaultEventHandler implements EventHandler {
+  constructor() {
+    this.mapHandlers = {
+      click: () => {},
```

Comment:
Some of these take event params with a certain type, how hard is it to add those here? I haven't played with `npm run check` yet to see what it thinks